worlds/alttp/Regions.py

```diff
@@ -19,347 +19,344 @@ def create_regions(world, player):
-        create_dungeon_region(player, 'Ganons Tower (Bottom)', 'Ganon\'s Tower', ['Ganons Tower - Bob\'s Chest', 'Ganons Tower - Big Chest', 'Ganons Tower - Big Key Room - Left',
+        create_dungeon_region(player, 'Ganons Tower (Bottom)', 'Ganon\'s Tower', world, ['Ganons Tower - Bob\'s Chest', 'Ganons Tower - Big Chest', 'Ganons Tower - Big Key Room - Left',
                                                                           'Ganons Tower - Big Key Room - Right', 'Ganons Tower - Big Key Chest']),
-        create_dungeon_region(player, 'Ganons Tower (Top)', 'Ganon\'s Tower', None, ['Ganons Tower Torch Rooms']),
-        create_dungeon_region(player, 'Ganons Tower (Before Moldorm)', 'Ganon\'s Tower', ['Ganons Tower - Mini Helmasaur Room - Left', 'Ganons Tower - Mini Helmasaur Room - Right',
+        create_dungeon_region(player, 'Ganons Tower (Top)', 'Ganon\'s Tower', world, None, ['Ganons Tower Torch Rooms']),
+        create_dungeon_region(player, 'Ganons Tower (Before Moldorm)', 'Ganon\'s Tower', world, ['Ganons Tower - Mini Helmasaur Room - Left', 'Ganons Tower - Mini Helmasaur Room - Right',
                                                                                   'Ganons Tower - Pre-Moldorm Chest'], ['Ganons Tower Moldorm Door']),
-        create_dungeon_region(player, 'Ganons Tower (Moldorm)', 'Ganon\'s Tower', None, ['Ganons Tower Moldorm Gap']),
-        create_dungeon_region(player, 'Agahnim 2', 'Ganon\'s Tower', ['Ganons Tower - Validation Chest', 'Agahnim 2'], None),
-        create_cave_region(player, 'Pyramid', 'a drop\'s exit', ['Ganon'], ['Ganon Drop']),
-        create_cave_region(player, 'Bottom of Pyramid', 'a drop\'s exit', None, ['Pyramid Exit']),
-        create_dw_region(player, 'Pyramid Ledge', None, ['Pyramid Entrance', 'Pyramid Drop']),
-        create_lw_region(player, 'Desert Northern Cliffs'),
-        create_dw_region(player, 'Dark Death Mountain Bunny Descent Area')
+        create_dungeon_region(player, 'Ganons Tower (Moldorm)', 'Ganon\'s Tower', world, None, ['Ganons Tower Moldorm Gap']),
+        create_dungeon_region(player, 'Agahnim 2', 'Ganon\'s Tower', world, ['Ganons Tower - Validation Chest', 'Agahnim 2'], None),
+        create_cave_region(player, 'Pyramid', 'a drop\'s exit', world, ['Ganon'], ['Ganon Drop']),
+        create_cave_region(player, 'Bottom of Pyramid', 'a drop\'s exit', world, None, ['Pyramid Exit']),
+        create_dw_region(player, 'Pyramid Ledge', world, None, ['Pyramid Entrance', 'Pyramid Drop']),
+        create_lw_region(player, 'Desert Northern Cliffs', world),
+        create_dw_region(player, 'Dark Death Mountain Bunny Descent Area', world)
     ]
 
     world.initialize_regions()
 
 
-def create_lw_region(player: int, name: str, locations=None, exits=None):
-    return _create_region(player, name, RegionType.LightWorld, 'Light World', locations, exits)
+def create_lw_region(player: int, name: str, world: MultiWorld, locations=None, exits=None):
```

Comment:
I would expect the signature of these to be (world, player, name, *, **) I think that's an order already established somewhere.

Reply:
You can reorder them project wide withe PyCharm refactor btw, don't do that by hand.